tests/vmi_multus_test.go

```diff
@@ -1049,3 +1053,21 @@ func checkSriovEnabled(virtClient kubecli.KubevirtClient, sriovResourceName stri
 	}
 	return false
 }
+
+func findSRIOVInterface(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, networkName string) string {
+	var interfaceName string
+	EventuallyWithOffset(1, func() string {
```

Comment:
why do we eventually this?

Reply:
virt-launcher is doing a polling each 2 minutes, in order to request interface list from qemu guest agent,
since it does not working in polling mode, it might take some time until the next report request happens.

Reply:
Removed it since we just need the name according mac, and those are already reported because they were part of the scheme and not part of cloud-init or on the fly which can cause time until its reported.